Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM python:3.10 AS base
 LABEL org.wayscript.image.authors="founders@wayscript.com"
 
-RUN apt-get update && apt-get -yu dist-upgrade
+RUN apt-get update && apt-get -yu dist-upgrade freetds-dev
```

Comment:
Is this change related to the new function?

Reply:
This change was made to fix a currently existing build error that has occurred likely due to remote dependency changes